lxd/instance/drivers/driver_lxc.go

```diff
@@ -7913,8 +7938,46 @@ func (d *lxc) removeDiskDevices() error {
 	return nil
 }
 
-// Network I/O limits.
-func (d *lxc) setNetworkPriority() error {
+func setNetworkPriorityWithNetfilter(d *lxc, networkPriority int) error {
+	// in setNetworkPriorityWithCgroup we set skb->priority on all
```

Comment:
Wouldn't parts of this comment be better placed on top of the func?

Reply:
sure, will do.